util/finding.go

```diff
@@ -57,6 +58,11 @@ func IsValidFinding(finding Finding) bool {
 		// if the source and sink are the same, return false and do not print out the finding
 		return false
 	}
+	// add filtering for findings with chan sources
+	if strings.Contains(finding.Untrusted_Source[0].SourceCode, "make(chan") {
+		log.Printf("Filtering Finding for Source: %s\n", finding.Untrusted_Source[0].SourceCode)
```

Comment:
let's configure this logging statement to only output if the debug flag is passed

Reply:
👍